mmv1/products/vertexai/api.yaml

```diff
@@ -151,8 +151,7 @@ objects:
     properties:
       - !ruby/object:Api::Type::String
         name: name
-        description: Output only. The resource name of the Endpoint.
-        output: true
+        description: The resource name of the Endpoint.
```

Comment:
This can be input: true & url_param_only & required now

Reply:
And name_from_self_link can probably be removed from terraform.yaml

Reply:
Done.